src/pages/club-overview/club-overview.mdx

```diff
@@ -20,7 +20,12 @@ In short our mission is:
 
 Our project meetings happen **every Sunday from 12:00 PM - 3:00 PM**. We start each meeting with some casual conversation and discuss the broader direction of the project. From here we define next steps and split off into teams to pair program. Since everything in the project is somehow connected, it is very common for developers to cross over between groups to help with integrations or seek answers to complex problems.
 
-Guest lecturers speak at the club on an approximately montly basis. We've had the oppurutuniy to converse with the Amazon Alexa team, Project Jupyter and several smaller AI startups. This is a great oppurunity to ask burning questions about the design strategies of these amazing projects and apply to job postings.
+Guest lecturers speak at the club on an approximately monthly basis. We've had the opportunity to converse with the Amazon Alexa team, Project Jupyter and several smaller AI startups. This is a great opportunity to ask burning questions about the design strategies of these amazing projects and apply to job postings.
+
+## Upcoming Events
+
+- Google: We'll be hosting a guest speaker from Google who will be coming to talk to us on October 22nd! He'll be talking about his work and how he got there, and will open up the floor for any questions or casual discussion afterwards.
```

Comment:
I'd add the dates/times to the beginning, like 
"Google, 10/22, 11AM - 12 PM:"
"Microsoft, 11/05, 11AM - 12 PM:"

And a note after the two points saying that virtual meeting links will be posted in Slack

Reply:
Thanks for adding these updates!